README
# Dakosy Reservations

Verwaltet Buchungen für Seminare des Kunden DAKOSY.
Erweitert die TYPO3-Erweiterung t3events.


ChangeLog
2023-08-09         [RELEASE] Release of t3events_course 1.0.0 (Steffen Mevius)$
2023-06-23 a842c2a [TASK] change pagination default (Steffen Mevius)
2023-06-19 e86e214 [TASK][43687] Provide setting 'module' for rendering backend module (Oliver Noth)
2023-06-14 b013cc3 [TASK] add pagination (Steffen Mevius)
2023-06-01 42cdce3 [TASK] fix const (Steffen Mevius)
2023-05-31 1aa596d [TASK] execute rector, add vendorExtensionName (Steffen Mevius)
2023-05-31 1f83e49 [TASK] add v11 compatibility to composer.json (Steffen Mevius)
2021-09-07 9998a99 [BUGFIX] enable richtext  in field requirements (Timo Gockel)
2021-08-26 c1633ac [TASK] Remove call to parent construct in CourseController (Santiago Nunez Negrillo)
2021-08-12 b493a93 [TASK] Adjust compatVersion condition in TypoScript (Santiago Nunez Negrillo)
2021-08-12 2c4b2ed [TASK] Register extension icons (Santiago Nunez Negrillo)
2021-08-05 7fe53a8 [TASK] Adjust templates to transform deprecated uploads fields to FAL fields (Santiago Nunez Negrillo)
2021-08-04 e3b452a [TASK] Adjust TCA configuration for TYPO3 v10 (Santiago Nunez Negrillo)
2021-08-04 0fd785a [TASK] Set examCosts field definition to float --> double is not an Extbase valid type (Santiago Nunez Negrillo)
2021-08-04 36855b4 [TASK] Change Extbase persistence configuration to TYPO3 v10 syntax (Santiago Nunez Negrillo)
2021-07-21 b6f0559 [TASK] Set version of dwenzel/t3events to dev-feature/version10compatible (Santiago Nunez Negrillo)
2021-07-21 4cc6aac [TASK] Change use of deprecated constants TYPO3_version (Santiago Nunez Negrillo)
2021-07-21 79d4aba [TASK] Move registerPlugin function to TCA/Overrides/tt_content.php (Santiago Nunez Negrillo)
2021-07-21 968c77b [TASK] Add Services.yaml configuration file (Santiago Nunez Negrillo)
2021-07-21 935d106 [TASK] Move TypoScript static file include to TCA/Overrides/sys_template.php file (Santiago Nunez Negrillo)
2021-07-21 878d12b [TASK] Change use of old @lazy and @inject annotations (Santiago Nunez Negrillo)
2021-07-21 de21007 [TASK] Adjust plugin and module configuration for TYPO3 v10 (Santiago Nunez Negrillo)
2021-07-15 667ff30 [TASK] Adjust composer.json (Santiago Nunez Negrillo)
2021-07-15 fc61ac2 [TASK] Update composer.json and ext_emconf.php files for TYPO3 v10 (Santiago Nunez Negrillo)
2019-08-14 79382b0 [RELEASE] Release of t3events_course 0.11.0 (Dirk Wenzel)
2019-08-14 ed032f3 [CI] require nimut/testing-framework: ^3.0 || ^4.0 (Dirk Wenzel)
2019-08-14 82a8257 [CI] include mysql service (Dirk Wenzel)
2019-08-14 1662114 [TASK] dependencies updated - support for PHP < 7.2 dropped - allow TYPO3 8 and 9 (Dirk Wenzel)
2019-08-14 1814311 [TASK] require dwenzel/t3extension-tools:~1.3 (Dirk Wenzel)
2019-08-13 cf38ceb [TASK] extension configuration refactored - require dwenzel/t3events:~1.1 - require dwenzel/t3extension-tools:~1.2 (Dirk Wenzel)
2019-08-13 b849472 [TASK] use canonical name of main module for registration (Dirk Wenzel)
2018-09-07 21849d5 [RELEASE] Release of t3events_course 0.10.0 (Dirk Wenzel) (Dirk Wenzel)
2018-07-06 4a38caa [TASK] CI: use typo3/minimal instead of typo3/core (Dirk Wenzel)
2018-07-06 dadacf9 [TASK] require dwenzel/t3events:>=0.40 (Dirk Wenzel)
2018-07-06 012661c [BUGFIX] fixed error caused by missing settings in backend controllers. Parent class ActionController now implements method injectConfigurationManager and fetches the settings there. Therefore we can not use our own implementation anymore. BackendControllers now provide the configuration manager via getter method. (Dirk Wenzel)
2018-06-28 a1a304a [TASK] CourseBackendController extended - now uses BackendTemplateView - method newAction added (Dirk Wenzel)
2018-06-28 6c79421 [TASK] CourseBackendController and ScheduleBackendController refactored using traits. - obsolete method createDemandFromSettings removed. (Dirk Wenzel)
2018-06-27 208f2db [RELEASE] Release of t3events_course 0.9.0 (Dirk Wenzel) (Dirk Wenzel)
2018-06-27 8fb3563 [BUGFIX] fixed error in controllers caused by invalid count() usage (under PHP 7.2) (Dirk Wenzel)
2018-06-27 8e814dc [BUGFIX] property extbaseType removed from CourseEventTrait. This avoids collision under PHP 7.2 (Dirk Wenzel)
2018-06-27 7a9bc67 [BUGFIX] failing unit tests fixed. (Dirk Wenzel)
2018-06-27 0ac0238 [BUGFIX] fix unit test for ScheduleDemandFactory - matcher for DateTimeObjects respects microseconds (Dirk Wenzel)
2018-06-27 e4465ed [BUGFIX] require-dev nimut/testing-framework:^3.0 (Dirk Wenzel)
2018-06-23 a59bf4d [TASK] require typo3/cms-core:>=8.7.0,<=8.7.99 and dwenzel/t3events:^0.39 (Support for TYPO3 version 6.2 and 7.6 dropped by required extension t3events) - travis CI build matrix adapted (Dirk Wenzel)
2018-06-23 1f96a5b [TASK] ScheduleBackendController now uses a BackendTemplateView (Dirk Wenzel)
2018-06-21 13e8d08 [TASK] require typo3/cms-core instead of deprecated package typo3/cms (Dirk Wenzel)
2018-06-19 558e520 [BUGFIX] fixed errors caused by missing module name for backend links and invalid attribute renderMode in flash messages viewhelper (Dirk Wenzel)
2018-06-19 8974d81 [TASK] use base class from testing framework for functional tests (Dirk Wenzel)
2018-06-19 575c5e3 [BUGFIX] fix error caused by missing method emitSignal in CourseBackendController (Dirk Wenzel)
2017-08-02 3b71839 [RELEASE] Release of t3events_course 0.8.0 (Dirk Wenzel) (Dirk Wenzel)
2017-04-25 d1442f0 [BUGFIX] save unit test coverage file in folder .Build/log/coverage/Unit (Dirk Wenzel)
2017-04-25 8c6f708 [TASK] travis-ci: use custom UnitTests.xml instead of those from testing frame work (Dirk Wenzel)
2017-04-19 6bae961 [BUGFIX] travis-ci: fix path of configuration file for functional tests (Dirk Wenzel)
2017-04-19 a0d0966 [TASK] allow TYPO3 8.7.x (Dirk Wenzel)
2017-04-19 cd297bf [TASK] use nimut/testing-framework, drop coveralls reports (Dirk Wenzel)
2017-03-07 57edd6e [TASK] require-dev phpunit/phpunit:<=4.8.0,<5.4 (Dirk Wenzel)

ext_emconf.php
<?php

/***************************************************************
 * Extension Manager/Repository config file for ext "t3events_course".
 *
 * Auto generated 02-08-2017 13:30
 *
 * Manual updates:
 * Only the data in the array - everything else is removed by next
 * writing. "version" and "dependencies" must not be touched!
 ***************************************************************/

$EM_CONF[$_EXTKEY] = ['title' => 'Courses', 'description' => 'Manage courses and course schedules. Extends t3events ', 'category' => 'plugin', 'author' => 'Dirk Wenzel, Boerge Franck', 'author_email' => 't3events@gmx.de, franck@cps-it.de', 'state' => 'beta', 'uploadfolder' => '0', 'createDirs' => '', 'clearCacheOnLoad' => 0, 'version' => '1.0.0',
	['depends' =>
		['typo3' => '8.7.0-10.4.99', 't3events' => '1.1.0-0.0.0'], 'conflicts' =>
		[], 'suggests' =>
		[]]];

